build/build.sh

```diff
@@ -1,12 +1,18 @@
-set -o errexit
+#!/usr/bin/env bash
+
+## build.sh: compile manuscript outputs from content using Manubot and Pandoc
+
+set -o errexit \
+    -o nounset \
+    -o xtrace
```

Comment:
The suggestion to not set `xtrace` here and instead allow it optionally with `bash -x script.sh` makes sense to me.

Reply:
Changed in https://github.com/manubot/rootstock/pull/233/commits/da0b397784d27d259e532b6d79c35405f56673b6 along with adding `-o pipefail`. From these [GNU docs](https://www.gnu.org/software/bash/manual/html_node/Pipelines.html)

> The exit status of a pipeline is the exit status of the last command in the pipeline, unless the pipefail option is enabled (see The Set Builtin). If pipefail is enabled, the pipeline’s return status is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands exit successfully. If the reserved word ‘!’ precedes the pipeline, the exit status is the logical negation of the exit status as described above. The shell waits for all commands in the pipeline to terminate before returning a value.

So I am not sure why `pipefail` does not interfere with the error suppression in:

```
DOCKER_EXISTS="$(command -v docker || true)"
```

Reply:
`command -v docker || true` is a list of commands, not a pipeline.